npm/vite-plugin-cypress-esm/README.md

```diff
@@ -43,27 +43,42 @@ export default defineConfig({
 })
 ```
 
-### `ignoreList`
+### `ignoreModuleList`
 
-Some modules may be incompatible with Proxy-based implementation. The eventual goal is to support wrapping all modules in a Proxy to better facilitate testing. For now, if you run into any issues with a particular module, you can add it to the `ignoreList` like so:
+Some modules may be incompatible with Proxy-based implementation. The eventual goal is to support wrapping all modules in a Proxy to better facilitate testing. For now, if you run into any issues with a particular module, you can tell the plugin to skip it like so:
 
 ```ts
 CypressEsm({
-  ignoreList: ['react-router', 'react-router-dom']
+  ignoreModuleList: ['react-router', 'react-router-dom']
 })
 ```
 
 You can also use a glob, which uses [`picomatch`](https://github.com/micromatch/picomatch) internally:
 
 ```ts
 CypressEsm({
-  ignoreList: ['*react*']
+  ignoreModuleList: ['*react*']
 })
 ```
 
-This will exclude modules matching the supplied pattern from being processed by this plugin. It is important to note that the actual import of any matching module into other files/modules will still be processed unless those destinations are themselves excluded via the list.
+This will exclude modules matching the supplied pattern(s) from being processed by this plugin. It is important to note that the act of importing any matching module into other files/modules will still be processed unless those destinations are themselves excluded via the list, but those imports will receive the unaltered version.
 
-React is known to have some conflicts with the Proxy implementation that cause problems stubbing internal React functionality. Since it is unlikely you want to stub parts of React itself, it's a good idea to add it to the `ignoreList`.
+Certain third-party dependencies such as React are known to have some conflicts with the Proxy implementation that cause problems stubbing internal functionality. Since it is unlikely you want to stub parts of React itself, it's a good idea to add it to the `ignoreModuleList`.
+
+### `ignoreImportList`
```

Comment:
I wonder if `ignoreFileList` is better? I get this is more technically accurate, though (or is it)?

Reply:
It's really tricky - right now this is based off the import target string which could be a *relative* path, so it's possible to ignore certain uses of a module while the mocked version is still used other places where that module is imported provided the relative paths are different.

Wonder whether we should update this to try to resolve all relative import paths to project root so that this would be more like a "don't mock uses of this module" sort of behavior

Reply:
Yeah, knowing the right behavior is tricky here. I think we should ship as is, asap, so Murat / Extend can test it out, and figure out the fine details after we know it's working.